pages/science/reactors/cvodes.rst

```diff
@@ -0,0 +1,244 @@
+method (C++) or the :py:func:`Reactor.set_advance_limit` method (Python) can be
+used to set the maximum amount that a specified solution component can change
+between output times. For an example of this feature's use, see the example
+`reactor1.py </examples/python/reactors/reactor1.py.html>`__. However, as a tradeoff, 
+the time step sizes in the output data are no longer guaranteed to be uniform.
+
+Even though Cantera comes pre-defined with typical parameters for tolerances
+and the maximum internal time step, the solver may not be correctly configured
+for the specific system. In this case the internal timestep solutions may not 
+converge towards a single value. To solve this problem, three parameters can be 
+tuned: The absolute time stepping tolerances, the relative time stepping tolerances, 
+and the maximum time step. A reduction of the latter value is particularly useful 
+when dealing with abrupt changes in the boundary conditions (for example, 
+opening/closing valves; see also the `IC engine example </examples/python/reactors
+/ic_engine.py.html>`__).
+
+How Does Cantera's Reactor Network Time Integration Feature Actually Work?
+==========================================================================
+
+There's a great description of the science behind Cantera's reactor network 
+simulation capabilities available on the Cantera website, 
+`here <https://cantera.org/science/reactors/reactors.html>`__. This section will go into more 
+developer-oriented detail about how the last step, ``ReactorNet``'s 
+`time integration methods <https://cantera.org/science/reactors/reactors.html#time-
+integration-for-reactor-networks>`__, actually work. A ``ReactorNet`` object doesn't 
+perform time integration on its own. Cantera generates a Jacobian array for the set 
+of ``Reactor`` objects contained in the ``ReactorNet``. This Jacobian, along with 
+functions to evaluate the residual of each ODE representing the system, is passed to 
+an ``Integrator``.
+
+Following integration from Reactor Network creation to completion
+-----------------------------------------------------------------
+
+Step 1: Reactor(s) created in Reactor Network
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, let's take a look at a basic example to see how we might utilize Cantera's time integration 
+functionality. We'll simulate an isolated reactor in Python that is homogeneously filled by a gas 
+mixture (the gas state used in this example is arbitrary, but interesting because it's 
+explosive). Then we'll advance the simulation in time to an (arbitrary) absolute time of 
+1 second, noting the changes in the state of the gas.
+
+.. code-block::
+
+    >>> import cantera as ct                           #import Cantera's Python module
+    >>> gas = ct.Solution("gri30.yaml")                #create a default GRI-Mech 3.0 gas mixture
```

Comment:
I'd ditch the `>>>` prefixes, since this isn't intermingling inputs and outputs, and make sure that this gets formatted nicely with the correct syntax highlighting in the HTML output.